data/human/campaign events.txt

```diff
@@ -519,6 +519,80 @@ event "initial deployment 3"
 		fleet "Large Republic" 800
 		fleet "Navy Surveillance" 1200
 
+
+
+mission "Central Outfitters Expansion Trigger"
+	landing
+	invisible
+	to offer
+		has "event: initial deployment 2"
+	on offer
+		event "central outfitters expansion" 45
+
+event "central outfitters expansion"
+	planet "Earth"
+		add outfitter "Central Outfits Expansion"
+	planet "Luna"
+		add outfitter "Central Outfits Expansion"
+	planet "New China"
+		add outfitter "Central Outfits Expansion"
+	planet "New Wales"
+		add outfitter "Central Outfits Expansion"
+	planet "Heartland"
+		add outfitter "Central Outfits Expansion"
+
+mission "Central Outfitters Expansion"
+	outfitter
+	source
+		attributes "near earth" "dirt belt"
+		outfits "Tactical Scanner"
+		outfits "Volcano Afterburner"
+		outfits "Caldera Afterburner"
+		outfits "S3 Thermionic"
+		outfits "Heavy Rocket"
+		outfits "Meteor Missile"
+		outfits "Javelin"
+		outfits "Javelin Pod"
+		outfits "Javelin Mini Pod"
+		outfits "Javelin Storage Crate"
+		outfits "Small Radar Jammer"
+		outfits "Brig"
+		outfits "Liquid Nitrogen Cooler"
+	to offer
+		has "event: central outfitters expansion"
+		not "joined the free worlds"
+	on offer
+		conversation
+			`When you walk into <origin>'s spaceport outfitter you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. To your surprise, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```

Comment:
Should there be a comma here?
```suggestion
			`When you walk into <origin>'s spaceport outfitter, you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. To your surprise, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```
Calling it a "spaceport outfitter" also seems a little weird. Most places just call it the "outfitter."
The only place that says "spaceport outfitter" seems to be in reference to a Remnant one, and they seem to have a particular emphasis on vertical integration.
```suggestion
			`When you walk into <origin>'s outfitter you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. To your surprise, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```
```suggestion
			`When you walk into <origin>'s outfitter, you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. To your surprise, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```
Don't tell me it's to my surprise. Maybe I expected it.
```suggestion
			`When you walk into <origin>'s spaceport outfitter you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. Strangely, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```
```suggestion
			`When you walk into <origin>'s outfitter you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. Strangely, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```
```suggestion
			`When you walk into <origin>'s spaceport outfitter, you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. Strangely, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```
```suggestion
			`When you walk into <origin>'s outfitter, you notice it has a raft of new outfits for sale; cooling equipment, afterburners, even new generators. Strangely, almost all the new stock appears to be from Kraz Cybernetics and Delta V - companies you'd usually see selling along the Southern Rim, not far up here so close to the center of the Republic. It is especially strange as those companies have been vocal supporters of the Free Worlds secessionists.`
```

Reply:
Feels like it works with or without the comma to me. Fair point on "to your surprise"